src/routes/_surfaces/workspace/explore/time-range-constructor/timeRangeUtils.ts

```diff
@@ -0,0 +1,589 @@
+import {
```

Comment:
nit: I think we aggreed to kebab-case these file names back in May, so it should be `time-range-utils.ts`, similarly for all the others i this directory + their tests

Secondly, I believe we also suggested to keep the tests colocated with the files they're testing

Reply:
Ok yeah. It looks like we're still split on casing through the codebase, but I can switch this one to kebab.

Colocating these tests make a lot of sense. I didn't see any other test files colocated, but happy to blaze the trail with this one.